### ISSUE TITLE
[NTR] Acinar dysplasia 

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#3188]
**Preferred term label:**
Acinar dysplasia 

and children:

- AcDys (Acinar dysplasia) FGF10
- AcDys (Acinar dysplasia) FGFR2
- AcDys (Acinar dysplasia) TBX4

Related to #3026 
This is based on the discussion with @mgriese

@mgriese - I am wondering if these terms are actually diseases or phenotypes? We have acinar dysplasia in HPO:
https://hpo.jax.org/app/browse/term/HP:0033209

Another question @mgriese: is this congenital AcDys? Is there non-congenital AcDys?

I am not sure if we could add the specific gene variant terms to HPO, but we can ask Peter. 

If we add these to Mondo, Chris brought up the question of why these aren't in OMIM and if they should be added there. I'll wait for your response to whether these are diseases vs phenotypes before proceeding, thanks @mgriese

cc @kallia-p

**Comment by nicolevasilevsky** [comment]
related to https://github.com/monarch-initiative/mondo/pull/3147

**Comment by mgriese** [comment]
Phenotype is the histology, i.e. not distinguisable; that means, different genetic causes have same phenotype.

**Comment by nicolevasilevsky** [comment]
@ahamosh should acinar dysplasia be added to OMIM with the specific mutations listed above?

I see some papers describing this as a disease:

- https://onlinelibrary.wiley.com/doi/full/10.1111/jpc.13494_169
- https://pubmed.ncbi.nlm.nih.gov/15490144/ (I wonder if this is the same as alveolar capillary dysplasia?)
- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3699159

**Comment by ahamosh** [comment]
Getting to this now.  Acinar dysplasia is not the same as alveolar capillary dysplasia. OMIM only discusses acinar dysplasia in the context of pulmonary hypoplasia which is currently a null entry.
